snapcraft/extensions/kde_neon_6.py

```diff
@@ -309,5 +388,6 @@ def get_parts_snippet(self) -> Dict[str, Any]:
                 "source": str(source),
                 "plugin": "make",
                 "make-parameters": [f"PLATFORM_PLUG={self.kde_snaps.content_kf6}"],
```

Comment:
```suggestion
                "make-parameters": ["PLATFORM_PLUG=kf6"],
```

Reply:
Huh? That ids wrong

Reply:
That id isn't used anymore. It was used before `core22`. So, I created this new PR to use that id again

https://github.com/canonical/snapcraft/pull/5109/files#diff-3566809cfc275dce60b796248d656b480205f412d888fcf1b7d8be2cbd1f578cR3

Reply:
And with this, we don't need redundant command-chains.